README.md

```diff
@@ -104,30 +115,40 @@ Implementation is very easy and can be performed in just a few steps (assuming t
 * place dedicated Twig helper in place where you want to display recommendations:
 
 ```twig
-{{ yc_show_recommendations(content.id, 'scenario_name', limit, 'content_type', 'template',
-    ['ez_publishedDate', 'ez_url', 'title', 'image', 'author', 'intro']
+{{ yc_show_recommendations(content.id,
+    scenario = '',
+    limit = '',
+    contentType = '',
+    template = '',
+    attributes = []
 ) }}
 ```
 
-Parameter meanings:
+Parameter meanings (all bellow are required):
 
 Parameter       | Type   | Description
 ----------------|--------|------------
 `content.id`    | int    | this is in content based views normally the twig variable holding the content id (we want to get recommendations for)
```

Comment:
yes but the name of the parameter is wrong. this is mixing parameter name with the name of the variable used in the example. please fix.